src/runtime/node/tty/write-stream.ts

```diff
@@ -0,0 +1,38 @@
+import type tty from "node:tty";
+import { Socket } from "node:net";
+
+export class WriteStream extends Socket implements tty.WriteStream {
+  clearLine(dir: tty.Direction, callback?: (() => void) | undefined) {
+    return false;
+  }
+  clearScreenDown(callback?: (() => void) | undefined) {
+    return false;
+  }
+  cursorTo(
+    x: number,
+    y?: number | undefined,
+    callback?: (() => void) | undefined,
+  ): boolean;
+  cursorTo(x: number, callback: () => void): boolean;
+  cursorTo(x: unknown, y?: unknown, callback?: unknown): boolean {
+    return false;
+  }
+  moveCursor(dx: number, dy: number, callback?: (() => void) | undefined) {
+    return false;
+  }
+  getColorDepth(env?: object | undefined): number {
+    return 1;
+  }
+  hasColors(count?: number | undefined): boolean;
+  hasColors(env?: object | undefined): boolean;
+  hasColors(count: number, env?: object | undefined): boolean;
+  hasColors(count?: unknown, env?: unknown): boolean {
+    return false;
+  }
+  getWindowSize(): [number, number] {
+    return [0, 0];
```

Comment:
I wonder if 0 could be a problematic value. maybe not.

I tried to see what webcontainers do and they seem to error due to what looks like an internal error: https://stackblitz.com/edit/stackblitz-starters-ch491u?description=Starter%20project%20for%20Node.js,%20a%20JavaScript%20runtime%20built%20on%20Chrome%27s%20V8%20JavaScript%20engine&file=index.js,package.json&title=node.new%20Starter

// cc: @d3lm

Reply:
A mock value makes sense indeed if it is used for formatting / line wrapping in a lib. I think 80x24 commonly is used as fallback (i use 25 for OCD reasons haha!)

Reply:
i think we can use class props btw to generate value?)